openedx/adg/lms/utils/decorators.py

```diff
@@ -0,0 +1,26 @@
+"""
+A utility for custom decorators
+"""
+import functools
+
+from django.conf import settings
+from django.dispatch import receiver
+
+
+def suspendingreceiver(signal, **decorator_kwargs):
+    """
+    A custom decorator tests, which can suspend receiver by overriding the SUSPEND_RECEIVERS
+    setting per test method or class.
+    """
+
+    def our_wrapper(func):
```

Comment:
Can we please give it a meaningful name?

Reply:
done